### PR TITLE
Add Coordinated Vulnerability Disclosure details

### PR DESCRIPTION
**Pull request by pjbgf**
This is largely based on CNCF's embargo policies, with some influence from Kubernetes Private Distributors List.

References:
https://github.com/cncf/tag-security/tree/main/project-resources/templates
https://github.com/kubernetes/committee-security-response/blob/main/private-distributors-list.md

### CODE REVIEW

File: CVD_LIST.md
Comment (stefanprodan) [inline CVD_LIST.md:42]: How can I access the data from here, also how can I edit the form?

Comment (hiddeco) [reply CVD_LIST.md:42]: Be a listed member in https://github.com/fluxcd/.github/blob/main/SECURITY.md.

File: CVD_MESSAGE_TEMPLATE.md
Comment (hiddeco) [inline CVD_MESSAGE_TEMPLATE.md:67]:
```suggestion
```

File: CVD_LIST.md
Comment (scottrigby) [reply CVD_LIST.md:42]: I am listed there but not 

We should add these details (such as access to this google doc form) to an onboarding/offboarding policy to the security team description. @dholbach mentions this here https://github.com/fluxcd/community/issues/214#issuecomment-1195262379 And in fluxcd/community/GOVERNANCE.md

Daniel and I discussed we should list `core maintainers: adding a security team member` to the supermajority decisions list here https://github.com/fluxcd/community/blob/main/GOVERNANCE.md#supermajority-decisions

File: CVD_MESSAGE_TEMPLATE.md
Comment (scottrigby) [inline CVD_MESSAGE_TEMPLATE.md:67]: Updating link to match above

```suggestion
[embargo policy]: https://github.com/fluxcd/.github/CVD_LIST.md#embargo-policy
```

**Review by scottrigby — CHANGES_REQUESTED** [review]
Need to fix link, added code suggestion. Everything else looks great to me.

Request to join note is just for discussion, not blocking this PR.

File: CVD_MESSAGE_TEMPLATE.md
Comment (pjbgf) [reply CVD_MESSAGE_TEMPLATE.md]: @scottrigby you should have access to the file now.

**Review by hiddeco — APPROVED** [review]

**Review by scottrigby — APPROVED** [review]
🔐

**Review by stefanprodan — APPROVED** [review]

**Pull request merged**
